### PR TITLE
fix(folder) : #MAG-351 fix modal to be iso with explorer

### PR DESCRIPTION
**Pull request by FlorentMr**
## Describe your changes
fix modal CSS not being ISO enough of blog, mindmap, ...

## Checklist tests
Check if the visuals are the same as blog ones.

## Issue ticket number and link
[MAG-351](https://jira.support-ent.fr/browse/MAG-351)

## Checklist before requesting a review (magic string, indentation, comment/documentation...)

- [x] I have detailed the tests to do in my feature/fix in order to prevent consequents regressions (must specify in **Checklist tests**)
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (API Doc etc...) - (must specify in **Description** for target version)
- [ ] If it is a consequent feature, I have added thorough tests.
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been added to this project (must specify in **Description**)

### CODE REVIEW

File: frontend/src/components/create-folder/CreateFolder.tsx [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import React, { FunctionComponent, useEffect, useState } from "react";
```
Comment (alicedraillard) [inline frontend/src/components/create-folder/CreateFolder.tsx:1]: le nom de commit mdr
essaie plutot :
fix(folder) : #MAG-351 fix modal to be iso with explorer

File: frontend/src/components/create-folder/CreateFolder.tsx [hunk 2]
```diff
@@ -67,50 +69,49 @@ export const CreateFolder: FunctionComponent<props> = ({
...
-          size="lg"
           viewport={false}
         >
           <Modal.Header onModalClose={resetFields}>
-            {folderToUpdate != null ? (
-              <h4>{t("magneto.folder.rename")}</h4>
-            ) : (
-              <h4>{t("magneto.create.folder")}</h4>
+            {t(
+              folderToUpdate != null
```
Comment (ThomasSevagen) [inline frontend/src/components/create-folder/CreateFolder.tsx:76]: pas d'égalité stricte?

File: frontend/src/components/create-folder/CreateFolder.tsx [hunk 3]
```diff
@@ -67,50 +69,49 @@ export const CreateFolder: FunctionComponent<props> = ({
...
+              variant="ghost"
+              onClick={resetFields}
+            >
+              {t("magneto.cancel")}
+            </Button>
+            <Button
+              color="primary"
+              type="submit"
+              variant="filled"
+              disabled={title == ""}
```
Comment (ThomasSevagen) [inline frontend/src/components/create-folder/CreateFolder.tsx:110]: pas d'égalité stricte?

File: frontend/src/components/create-folder/CreateFolder.tsx (hunk 2 above)
Comment (FlorentMr) [reply frontend/src/components/create-folder/CreateFolder.tsx:77]: Ca permet également de s'assurer qu'on filtre les valeurs undefined, au moins on est safe (ya probablement plus compact que ça après j'imagine, avec un ? potentiellement)

File: frontend/src/components/create-folder/CreateFolder.tsx (hunk 1 above)
Comment (FlorentMr) [reply frontend/src/components/create-folder/CreateFolder.tsx:1]: C'est pas facile les fins de Vendredi.. j'ai changé, merce pour la propale)

**Review by ThomasSevagen — APPROVED** [review]

**Pull request merged**